plugin-flex-ts-template-v2/src/feature-library/send-audio-rec-file/flex-hooks/strings/index.ts

```diff
@@ -8,12 +8,20 @@ import zhHans from './zh-hans.json';
 export enum StringTemplates {
   AudioRecorder = 'PSAudioRecorder',
   RecordingMessage = 'PSRecordingMessage',
+  StartRecButton = 'StartRecButton',
+  StopRecButton = 'StopRecButton',
+  RemoveRecButton = 'RemoveRecButton',
+  MicBlockedMessage = 'MicBlockedMessage',
 }
 
 export const stringHook = () => ({
   'en-US': {
     [StringTemplates.AudioRecorder]: 'Audio Recorder',
     [StringTemplates.RecordingMessage]: 'Recording in progress ',
+    [StringTemplates.StartRecButton]: 'Start Recording',
+    [StringTemplates.StopRecButton]: 'Stop Recording',
+    [StringTemplates.RemoveRecButton]: 'Discard Recording',
+    [StringTemplates.MicBlockedMessage]: 'Allow this page to access your Microphone before use this feature.',
```

Comment:
`microphone` should be lower case, also change `use` -> `using`

Reply:
done